### ISSUE TITLE
new Validating in version 4 has error in typescript

### ISSUE THREAD

**Issue by perfeshnal** [shahabyazdi/react-multi-date-picker#186]
hi shahab yazdi 
thanks for this great component 
I receiving this error on new version, Please advise
```
Type '(date: any, { validatedValue, input, isTyping }: any) => void' is not assignable to type '((selectedDates: DateObject | DateObject[] | null) => false | void) & ((selectedDates: DateObject | DateObject[] | null, validatedValue: string | string[], input: HTMLElement, isTyping: boolean) => false | void)'.
  Type '(date: any, { validatedValue, input, isTyping }: any) => void' is not assignable to type '(selectedDates: DateObject | DateObject[] | null) => false | void'.ts(2322)
index.d.ts(187, 5): The expected type comes from property 'onChange' which is declared here on type 'IntrinsicAttributes & CalendarProps & DatePickerProps'
```

```
<Controller
      name={props.name!}
      control={control}
      rules={rules}
      render={({ field, fieldState: { error } }) => (
        <DatePicker
          format="YYYY٫MM٫DD"
          weekDays={["ش", "ی", "د", "س", "چ", "پ", "ج"]}
          onOpenPickNewDate={false}
          calendar={persian}
          locale={persian_fa}
          plugins={[weekends()]}
          maxDate={new DateObject({ calendar: persian }).toLastOfYear()}
          render={<MaterialInput name={props.name} label={label} error={error !== undefined} />}
          value={field.value || ""}
          onChange={(date: any, { validatedValue, input, isTyping }: any) => {
            field.onChange(date?.isValid ? date : null);
          }}
          containerStyle={{
            width: "100%",
          }}
          {...props}
        />
      )}
    />
```

**Comment by shahabyazdi** [comment]
I will work on it and let you know the result.

**Comment by SDAIA-N** [comment]
Any sign of hope ?

**Comment by shahabyazdi** [comment]
Hi @perfeshnal ,

Apologies for the delayed response.

Could you please try v4.3.3 and let me know if that resolved the issue?

**Comment by perfeshnal** [comment]
Thank you Shahab
problem is solved in version 4.3.3 and it works well 👍
